scripts/bft-single-node-test

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+### CONFIGURATION
+
+CLI=jcli
+NODE=jormungandr
+
+FAUCET_AMOUNT=1000000000
+ADDRTYPE="--testing"
+REST_PORT="8443"
+
+### MAKE EVERYTHING
+
+# faucet
+FAUCET_SK=$($CLI key generate --type=Ed25519Extended)
+FAUCET_PK=$(echo ${FAUCET_SK} | $CLI key to-public)
+FAUCET_ADDR=$($CLI address single ${ADDRTYPE} ${FAUCET_PK})

```

Comment:
make the faucet an account

Reply:
AFAIK accounts are not working currently (cannot get account state ) . See https://github.com/input-output-hk/jormungandr/issues/388

Reply:
they are working now